### ISSUE TITLE
Dropdown Support

### ISSUE THREAD

**Issue by astra-mobiliya** [Microsoft/PowerBI-visuals-ChicletSlicer#57]
Is there any plan to provide dropdown view support?
Your Forced selection feature and dropdown view will together make a great set of combination.

**Comment by iuriemaxim** [comment]
Indeed, this would make Chiclet slicer the most used slicer in PBI as no other slicer allow for forced selection. Chiclet with images already transform the PBI reports into an app. Dropdown will save a lot of space and will be able to design menus like in any other application.
So, any plan for dropdown support?

**Comment by helen508light** [comment]
We'll consider your suggestions in the next versions of the visual.
Please note that source code for the visual is public, feel free to fork and modify it for your needs.
